BSB_LAN/BSB_LAN.ino

```diff
@@ -9478,6 +9475,11 @@ void setup() {
     temp_bus_pins[1] = 18;
 #endif
   }
+
+  #if defined(__arm__)
+  pinMode(temp_bus_pins[0], INPUT); //RX-pin of hardware serial on Due
+  #endif
```

Comment:
Indention?

Reply:
In fact, pinMode(19, INPUT) is equivalent to pinMode (temp_bus_pins[0], INPUT) for Due, but allows us not to forget about the pin number when we make further changes.
Test it on my Due without problem.

Reply:
Sorry, I didn't mean "intention", but "indention" because it looks here that the `#if` is not left-aligned but indented?
Your change from 19 to the variable is really useful, of course!

Reply:
Actually, I'm not sure how we should deal if the indention of `#if` preprocessor directives. I think they should generally start at the beginning of the line because they have their own hierarchy. But I don't know if we should then start to indent them to the right when nesting starts or not. Maybe it's a good idea to indent them if they are nested, so we better see what depends on what. Could uncrustify do this, too?